index.html

```diff
@@ -282,12 +264,9 @@ <h2>Contact me</h2>
               required
             ></textarea>
             <button class="btn-submit" type="submit">Get connected</button>
```

Comment:
- [ ] Please fix the positioning of the `get connected` button by moving it to the left to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/187457752-637d07cc-f0bf-4833-98b1-1c42e89a57ef.png)  | ![image](https://user-images.githubusercontent.com/68796232/187457675-e03f80a8-0f54-4731-bb71-0285185d0005.png)  |

Reply:
In the "submit-btn" class I omitted "left-margin: 25%". This was my mistake. I placed it in the middle of the contact to look better.